Sources/Accessibility.swift

```diff
@@ -84,18 +84,4 @@ public enum Accessibility: Int, CustomStringConvertible, Equatable {
         
         return accessibilityAttribute as String
     }
-    
-    // MARK: Internal
-    
-    internal static func allValues() -> [Accessibility] {
```

Comment:
In order to maintain backwards compatibility with older Swift versions, we'll need to keep this method. However, we can deprecate it with a message that this property will be removed in a later version. Something like:
```
@available(swift, obsoleted: 4.2, renamed: "allCases", message: "This method should be removed in the next major version bump of Valet")
```

Technically we don't need to use deprecation syntax since this property is `internal`, but it can't hurt to use built-in Swift syntax to make our intent explicit.